contour/src/test/kotlin/com/squareup/contour/ContourTests.kt

```diff
@@ -376,4 +376,79 @@ class ContourTests {
     assertThat(otherView.width).isEqualTo(10 + 1)
     assertThat(otherView.height).isEqualTo(15 + 2)
   }
+
+  @Test
+  fun `using other axis width constraint`() {
```

Comment:
The formatting is a bit off here. @saket should we introduce an editor-config here?  Our formatting rules can be installed from [here](https://github.com/square/java-code-styles)

Reply:
Are you referring to `layoutBy` arguments indentation?